pkg/hypervisor/README.md

```diff
@@ -0,0 +1,23 @@
+# Config
+
+- `public_key` ([PubKey](#PubKey))
```

Comment:
Some anchors like this one seem broken. I'm not sure if it's possible to create correct anchors for such types. If not, what do you think about removing anchors for such types?